src/main/resources/matlab-bamboo-plugin.properties

```diff
@@ -5,14 +5,14 @@ matlab.plugin.command=Command
 plugin.command.error=Command field is empty.
 
 matlab.helpPath = Full path to MATLAB root folder
-matlab.options = Specify startup options
-matlab.options.field = Options
+matlab.options = Startup options
```

Comment:
I am not sure if we can remove "Specify" from the UI. In the initial plugin design with Preethi and Anjana, we decided that "Specify" is needed for clarity and for consistency with other verb phrases, such as **Disable** this task, in the task configuration panel.

Reply:
Ohh I see, sure I'll revert these changes